torchci/scripts/check_alerts.py

```diff
@@ -534,39 +467,31 @@ def check_for_recurrently_failing_jobs_alert(
         clear_alerts(existing_alerts, dry_run=dry_run)
         return
 
-    # In the current design, there should be at most one alert issue per repo and branch
-    assert len(existing_alerts) <= 1
-
-    if existing_alerts:
-        # New update, edit the current active alert
-        existing_issue = existing_alerts[0]
-        update_comment = gen_update_comment(existing_issue["body"], jobs_to_alert_on)
-
-        if update_comment:
-            new_issue = generate_failed_job_issue(
-                repo=repo, branch=branch, failed_jobs=jobs_to_alert_on
-            )
-            update_issue(new_issue, existing_issue, update_comment, dry_run=dry_run)
-        else:
-            print(f"No new change. Not updating any alert for {repo} {branch}")
-    else:
-        # No active alert exists, create a new one
+    if len(existing_alerts) == 0:
+        # Generate a blank issue if there are no issues so we can post an update
+        # comment, which will trigger a more informative workchat ping
         create_issue(
-            generate_failed_job_issue(
-                repo=repo, branch=branch, failed_jobs=jobs_to_alert_on
-            ),
-            dry_run=dry_run,
+            generate_failed_job_issue(repo=repo, branch=branch, failed_jobs=[]), dry_run
```

Comment:
As a new issue is created here, is there a quick way to get its ID right away instead of making another call to `fetch_alerts_filter`?

Reply:
done, but a bit sus